src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -0,0 +1,90 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+All the `deadline asset` commands:
+    * snapshot
+    * upload
+    * diff
+    * download
+"""
+
+import click
+
+from .._common import _handle_error
+
+
+@click.group(name="asset")
+@_handle_error
+def cli_asset():
+    """
+    Commands to work with AWS Deadline Cloud Job Attachments.
+    """
+
+
+@cli_asset.command(name="snapshot")
+@click.option("--root-dir", help="The root directory to snapshot. ")
+@click.option("--manifest-out-dir", help="Destination path to directory for created manifest. ")
+@click.option(
+    "--r",
```

Comment:
nit: For single letter flags, usually we do the single dash, like `-r`. Could we have both `-r` and `--recursive` here? Can also address this later.